person.rb

```diff
@@ -1,23 +1,33 @@
-class Person
+require_relative 'nameable'
+
+class Person < Nameable
+  attr_accessor :name, :age, :rentals
   attr_reader :id
-  attr_accessor :name, :age
 
-  def initialize(id, age, name = 'Unknown', parent_permission: true)
-    @id = id
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @id = Random.rand(1..1000)
+    @rentals = []
   end
 
-  private
+  def can_use_services?
+    of_age? || @parent_permission
+  end
 
-  def of_age?
-    @age >= 18
+  def correct_name
+    name
   end
 
-  public
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] Kindly do the same changes as i suggested in the `book` class to add a rental instance 👍